source/user-manual/capabilities/vulnerability-detection/offline-update.rst

```diff
@@ -265,7 +265,7 @@ Alternatively, the feeds can be loaded from a local path. To achieve it, configu
 
     <provider name="arch">
         <enabled>yes</enabled>
-        <path>/local_path/archlinux_all.json$</path>
+        <path>/local_path/archlinux_all\.json$</path>
```

Comment:
Testing the steps for Debian and Arch successfully populates the vulnerabilities database with feed information.
![image](https://user-images.githubusercontent.com/13010397/173858135-8b20fdad-3bc0-4223-9680-faa96bf18597.png)